Readme.md

```diff
@@ -4,24 +4,102 @@
+- Create a modern, user friendly interface with swipe-style navigation.
+
+## Implementation Details
+
+### Data Structure
+
+The data source contains a list of `Employee` objects, which is exposed by the `EmployeesData` view model through the `DataSource` property. The `Employee` class exposes employee-related properties (such as `FullName`, `Photo`, `JobTitle`, `EmailAddress`, and `City`). Images are stored as byte arrays and converted to BitmapImage objects at runtime.
+
+```csharp
+public class Employee {
+    public string FirstName { get; set; }
+    public string LastName { get; set; }
+    public string FullName => FirstName + " " + LastName;
+    public string JobTitle { get; set; }
+    public string EmailAddress { get; set; }
+    public string City { get; set; }
+    public byte[] ImageData { get; set; }
+
+    [XmlIgnore]
+    public BitmapImage Photo => imageSource ??= LoadImage();
+
+    BitmapImage LoadImage() {
+        var stream = new MemoryStream(ImageData);
+        var image = new BitmapImage();
+        image.BeginInit();
+        image.StreamSource = stream;
+        image.EndInit();
+        return image;
+    }
+}
+```
+
+### Templates
+
+The example defines two templates:
+
+* [`ItemHeaderTemplate`](https://docs.devexpress.com/WPF/DevExpress.Xpf.WindowsUI.SlideView.ItemHeaderTemplate) displays the employee’s first name.
+* [`ItemContentTemplate`](https://docs.devexpress.com/WPF/DevExpress.Xpf.WindowsUI.SlideView.ItemHeaderTemplate) displays detailed information, including photo, contact details, and job data.
+
+```xaml
+<DataTemplate x:Key="ItemHeaderTemplate">
+    <TextBlock Text="{Binding FirstName}" />
+</DataTemplate>
+
+<DataTemplate x:Key="ItemContentTemplate">
+    <Grid>
+        <!-- Photo, FullName, JobTitle, etc. -->
+    </Grid>
+</DataTemplate>
+```
+
+### Slide View Configuration
+
+```xaml
+<dxwui:SlideView
+    Header="Slide View"
+    ItemsSource="{Binding DataSource}"
+    ItemHeaderTemplate="{StaticResource ItemHeaderTemplate}"
+    ItemTemplate="{StaticResource ItemContentTemplate}" />
+```
+
+## Files to Review
 
 * [DataStorage.cs](./CS/SlideViewSample/DataStorage.cs)
-* **[MainWindow.xaml](./CS/SlideViewSample/MainWindow.xaml)**
-<!-- default file list end -->
-# WPF SlideView - Display a horizontal list of items with slide navigation
+* [MainWindow.xaml](./CS/SlideViewSample/MainWindow.xaml)
 
+## Documentation
 
-<p>This example demonstrates how to create a SlideView, bind it to data and use templates to visualize its items. </p>
+* [SlideView](https://docs.devexpress.com/WPF/DevExpress.Xpf.WindowsUI.SlideView)
+* [SlideViewItem](https://docs.devexpress.com/WPF/DevExpress.Xpf.WindowsUI.SlideViewItem)
+* [ItemHeaderTemplate](https://docs.devexpress.com/WPF/DevExpress.Xpf.WindowsUI.SlideView.ItemHeaderTemplate)
+* [ItemContentTemplate](https://docs.devexpress.com/WPF/DevExpress.Xpf.WindowsUI.SlideView.ItemHeaderTemplate)

```

Comment:
I think the URL is incorrect here. Can you please check?
Also, please use the full name of the help topic, like I did in my suggestions above.

Reply:
Thanks!